public/app/percona/add-instance/components/AzureDiscovery/Discovery.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import Discovery from './Discovery';
+
+jest.mock('app/percona/shared/helpers/notification-manager');
```

Comment:
This line can be removed, this helper is not in shared anymore.

Reply:
Done